packages/core-mobile/app/services/fcm/FCMService.ts

```diff
@@ -49,11 +57,14 @@ class FCMService {
         url: `${PROTOCOLS.CORE}://${ACTIONS.OpenChainPortfolio}`
       }
       await NotificationsService.displayNotification({
-        channelId: ChannelId.BALANCE_CHANGES,
         title: result.data.notification.title,
         body: result.data.notification.body,
-        data
-      })
+        data,
+        sound: iOSSoundNotification,
+        channelId:
+          result.data.notification.android?.channelId ??
+          ChannelId.BALANCE_CHANGES
```

Comment:
looks like you added the logic to allow undefined channelId in `NotificationsService.displayNotification`. is this defaulting to ChannelId.BALANCE_CHANGES logic still needed?

Reply:
Corrected in [Make displayNotification#channelId optional](https://github.com/ava-labs/core-mobile/pull/2126/commits/77d493028626805de126cd4ff6df054454a738d5)